services/speechsynthesis.md

```diff
@@ -0,0 +1,43 @@
+#  Speech synthesis
+
+     identifier: 0x1204d995
+
+A speech synthesizer
+
+## Registers
+
+    ro speaking: bool @ reading
+
+Reports if the engine is speaking.
+
+    rw enabled: bool @ intensity
+
+Determines if the speech engine is in a non-paused state.
+
+    rw lang?: string @ 0x80
+
+Default language used for utterances.
+
+    rw volume?: u8 / @ 0x81
+
+Default volume for utterances
+
+    rw pitch?: u8 / @ 0x82
```

Comment:
should specify default for pitch

Reply:
somehow ``rw pitch? = 0.5:`` ... does not parse